tendermint/tests/support/lite/single_step_sequential/commit_tests.json

```diff
@@ -1976,6 +1975,6 @@
  				}
  			],
  			"expected_output": "no error"
- 		}
- 	]
- }
\ No newline at end of file
+		}
+	]
+}
```

Comment:
Were these re-generated, or is this because you used a different IDE and reformatted these?

Reply:
yep I switched to using a different IDE and it showed this diff. Haven't re-generated